controller_manager/src/controller_manager.cpp

```diff
@@ -821,6 +1277,69 @@ void ControllerManager::stop_controllers()
   }
 }
 
+void ControllerManager::switch_chained_mode()
+{
+  std::vector<ControllerSpec> & rt_controller_list =
```

Comment:
The list is not available in the caller method. But it could be. I would then also extend `stop_controllers` and `start_controllers` to get the list as parameter. We probably expect that one `manage_switch` is anyway, working with the same list.

Reply:
Done in fe20955